### PR TITLE
convert JS files to TS

### PR DESCRIPTION
**Pull request by bhavesh-desai-scratch**
- At some point, we need to run TS through Babel
- Upgraded to TS 3.0.1
- We should now import styled via rt-theme to get full typing

### CODE REVIEW

File: src/client/src/ui/StatusBar/styled.tsx [hunk 1]
```diff
@@ -88,10 +84,4 @@ export const ServiceRoot = styled.div`
   background-color: ${({ index = 0, theme }) =>
     // it'd be nice if this were selected from the original palette …
     darken(index / 50, theme.backgroundColor)};
-
-  ${Icon} {
```
Comment (evanrs) [inline src/client/src/ui/StatusBar/styled.tsx]: Don't remove functionality — provide the solution.

Additionally it is on the docket to get support for component selectors in TS — the better approach here is to make note of that.

File: src/client/src/ui/StatusBar/StatusBar.tsx [hunk 2]
```diff
@@ -1,16 +1,27 @@
...
-import { ThemeProvider } from 'rt-theme'
+import { styled, ThemeProvider } from 'rt-theme'
 
+import { GlobalState } from 'combineReducers'
 import Icon from './Icon'
-import { Root, Body, Fill, ExpandToggle, ServiceList, ServiceRoot, ServiceName, NodeCount } from './styled.js'
+import { Body, ExpandToggle, Fill, NodeCount, Root, ServiceList, ServiceName, ServiceRoot } from './styled'
 
 export const SERVICES = ['blotter', 'reference', 'execution', 'pricing', 'analytics'].map(serviceType => ({
-  serviceType
```
Comment (evanrs) [inline src/client/src/ui/StatusBar/StatusBar.tsx]: You've broken functionality — remove this change.

Or, revisit the use to understand the change you'd like to make.

The correct value for these keys is `null`.

Comment (bhavesh-desai-scratch) [reply src/client/src/ui/StatusBar/StatusBar.tsx]: What does it break?  They had a value of undefined before this change.

File: src/client/src/ui/StatusBar/styled.tsx (hunk 1 above)
Comment (bhavesh-desai-scratch) [reply src/client/src/ui/StatusBar/styled.tsx]: I have not removed this functionality. We will get TS running through babel at a later point and we can go back to doing things this way. I have added the margins back in the status bar file.

File: src/client/src/ui/StatusBar/StatusBar.tsx (hunk 2 above)
Comment (bhavesh-desai-scratch) [reply src/client/src/ui/StatusBar/StatusBar.tsx]: isConnected = null | true | false 

makes no sense

**Pull request merged**